### PR TITLE
Closes #131 sanitize price before saving

### LINKED ISSUE #131: sanitize price before saving of item
Entering a price with commas will break the site, so we should probably add a before save action to remove anything that isn't a number.

### PR DESCRIPTION
**Pull request by Tman22**
This allows ability for artist and admin to create items and to have commas and dollar signs into the form for price.

### CODE REVIEW

File: test/integration/artist_can_add_an_item_test.rb [hunk 1]
```diff
@@ -38,4 +38,30 @@ class ArtistCanAddAnItemTest < ActionDispatch::IntegrationTest
...
+    fill_in "Price", with: "$100,000"
+    fill_in "Description", with: "Salad? That's what my food eats"
+    select "#{category.name}", from: "item_category_id"
+    click_on "Create Item"
+
+    assert_equal item_path(Item.last), current_path
+    assert_equal 1, Item.count
+    assert page.has_content? Item.last.price
+    assert_equal artist, Item.last.user
+    assert_equal 100000, Item.last.price
```
Comment (Tman22) [inline test/integration/artist_can_add_an_item_test.rb]: Ignoring this

File: test/integration/admin_user_creates_item_test.rb [hunk 2]
```diff
@@ -26,4 +26,30 @@ class AdminUserCreatesItemTest < ActionDispatch::IntegrationTest
...
+    fill_in "Price", with: "$100,000"
+    fill_in "Description", with: "Salad? That's what my food eats"
+    select "#{category1.name}", from: "item_category_id"
+    select "#{artist.first_name}", from: "item_user_id"
+    click_on "Create Item"
+
+    assert_equal item_path(Item.last), current_path
+    assert_equal 1, Item.count
+    assert page.has_content? Item.last.price
+    assert_equal 100000, Item.last.price
```
Comment (Tman22) [inline test/integration/admin_user_creates_item_test.rb]: Ignoring this

File: test/integration/artist_can_add_an_item_test.rb (hunk 1 above)
Comment (ToniRib) [inline test/integration/artist_can_add_an_item_test.rb]: Agreed!

File: test/integration/admin_user_creates_item_test.rb (hunk 2 above)
Comment (ToniRib) [inline test/integration/admin_user_creates_item_test.rb]: Agreed!

**Comment by ToniRib** [comment]
Lookin' good, @Tman22!! I merged in master and am going to close this.

**Comment by Tman22** [comment]
Ready to go!

**Pull request merged**